dnstap/main.go

```diff
@@ -22,8 +22,9 @@ import "log"
 import "os"
 import "os/signal"
 import "runtime"
+import "syscall"
 
-import dnstap "github.com/dnstap/golang-dnstap"
+import "golang-dnstap"
```

Comment:
I wasn't sure what the reason was for specifying the repo itself here from within it, but in order to test this it needed to be changed to use the local code directly.